services/simcore/docker-compose.yml

```diff
@@ -396,7 +396,7 @@ services:
     deploy:
       update_config:
         parallelism: 2
-        order: start-first
+        order: stop-first
```

Comment:
i wonder how does this coordinate with the rest of the services that follow `start-first` policy. What happens if one of the latter  (i.e. newly started) communicates with the directorv2 (being stopped)

Q: what does `parallelism:2` mean in this context?

Reply:
- If they try to communicate with the service. Clearly they will get an error. It's up to the other services to deal with it. I will remind that currently `director-v2` is not restartable.
- Restart parallelism is related to the number of copies that are allowed to start at a give time while the service is being scaled. Since we only have one copy this is OK.

Reply:
- Did you check that the other services "deal with it"?
- is it then necessary?

Reply:
- The other services might not deal with it. But it's OK since this will happen during maintenance or by operator intervention.
- Yes avoid issues with director-v2 instances possibly doing the same operation twice.